truss/remote/baseten/rest_client.py

```diff
@@ -0,0 +1,39 @@
+from typing import Any, Dict
+
+import requests
+
+
+class RestAPIClient:
+    base_url: str
+    headers: Dict[str, str]
+
+    def __init__(self, base_url: str, headers: Dict[str, str]):
+        self.base_url = base_url
+        self.headers = headers
+
+    def _handle_error(self, resp: requests.Response):
+        if 400 <= resp.status_code < 500:
```

Comment:
Intentionally don't want to print out 5xx, since we might be showing too much information to the user